terraform/examples/daos_cluster/variables.tf

```diff
@@ -156,9 +156,15 @@ variable "server_pools" {
   description = "If provided, this module will generate a script to create a list of pools. pool attributes have to be specified in a format acceptable by [dmg](https://docs.daos.io/v2.0/admin/pool_operations/) and daos."
   default     = []
   type = list(object({
-    pool_name  = string
-    pool_size  = string
-    containers = list(string)
+    label = string
+    size  = string
+    tier_ratio = string
+    properties = string
+    containers = list(object({
+      label = string
+      type = string
+      properties = string
+      }))
```

Comment:
@lsitkiew  Please leave these types of changes to the GCPDAOS83  ticket.

Reply:
Ok, I will move this changes to other PR.